webservices/server-integration/pom.xml

```diff
@@ -42,102 +42,107 @@
     <name>WildFly: Web Services Server Integration Subsystem</name>
 
     <dependencies>
-    <dependency>
-       <groupId>junit</groupId>
-       <artifactId>junit</artifactId>
-       <scope>test</scope>
-    </dependency>
-
-    <dependency>
-       <groupId>org.jboss.msc</groupId>
-       <artifactId>jboss-msc</artifactId>
-    </dependency>
-
-    <dependency>
-       <groupId>org.wildfly.security</groupId>
-       <artifactId>wildfly-elytron</artifactId>
-    </dependency>
-
-    <dependency>
-       <groupId>org.wildfly.core</groupId>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
```

Comment:
Only mockito-core was added, rest of the lines were just reformatted...

Reply:
@TomasHofman  Can you please only add the  mockito-core dependency in this PR and format this pom file in another PR ?

Reply:
OK, done.